apps/finance/app/src/lib/utils.js

```diff
@@ -28,3 +28,8 @@ export function formatTokenAmount(
     formatDecimals(round(amount / Math.pow(10, decimals), rounding), 18)
   )
 }
+
+// Thousands Separator
+export function separateThousands(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
This doesn't work if the string has a decimal, e.g. (`'1.23456'`).

We should do the replace on only the whole number ahead of the decimal.

Reply:
Okay, I just made a new commit that should fix this.